backend/src/nodes/nodes/rest/__init__.py

```diff
@@ -0,0 +1,8 @@
+from ...category import Category
+
+category = Category(
+    name="REST",
+    description="Interact with external components via REST",
```

Comment:
"Stable Diffusion (External)" maybe?

Reply:
I do like the idea of an external category with the specific service as a subcategory.  We could call it "Automatic1111" instead of "Stable Diffusion", just to make that relationship more explicit.

Reply:
Or maybe "Automatic1111 Stable Diffusion"? I feel like SD should be referenced somewhere

Reply:
How about: category "Stable Diffusion (External)" subcategory "Automatic1111"

Reply:
I guess a question is whether there's a connection between categories and dependencies.  Right now, it looks like it's 1-to-1.  Installing PyTorch unlocks the entire PyTorch category.

Reply:
Yeah I'd say go with that. Makes enough sense.